modules/vactory_content_sheets/src/Commands/Commands.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Drupal\vactory_content_sheets\Commands;
+
+use Drush\Commands\DrushCommands;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+
+/**
+ * Vactory migrate drush commands.
+ */
+class Commands extends DrushCommands {
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * Commands construct definition.
+   */
+  public function __construct(EntityTypeManagerInterface $entityTypeManager) {
+    $this->entityTypeManager = $entityTypeManager;
+  }
+
+  /**
+   * Disable content sheets data.
+   *
+   * @command vactory-disbale-content-sheets
+   * @aliases vdcs
+   */
+  public function disableContentSheets() {
+    $this->output()->writeln('Process of disable content sheets start...');
+    $storage = $this->entityTypeManager->getStorage('node');
+    try {
+      $query = $storage->getQuery()
+        ->condition('type', 'vactory_page')
```

Comment:
@khalidbouhouch77 plz, add a pager here and batch on it, take inspiration from this https://www.drupal.org/project/drupal/issues/2577417

Reply:
Does this work with blocks ? can we specify a list of content types to update ?

Reply:
@Bahlaouane-Hamza 
- Done pour les block content.
- De plus, maintenant vous pouvez mentionner la liste des types de contenu à mettre à jour comme suit : **drush vdcs --bundles="vactory_page,vactory_news"**